src/components/CountriesList/CountriesList.tsx

```diff
@@ -1,5 +1,41 @@
+import { useState, useEffect } from "react";
 import { StyledDisplaySection } from "./CountriesList.styles";
+import api from "../../api/countriesApi";
+import { CountriesProps } from "./CountriesProps";
+import { SingleCountry } from "../singleCountry/SingleCountry";
 
-export const CountriesList = () => (
-	<StyledDisplaySection>CountriesList</StyledDisplaySection>
-);
+export const CountriesList = () => {
+	const [error, setError] = useState<boolean>();
+	const [countries, setCountries] = useState<CountriesProps[]>();
+
+	useEffect(() => {
```

Comment:
Make a note to yourself to refactor the in the future to extract this logic to some different place

Reply:
I moved fetching api logic to custom hook.